config.go

```diff
@@ -251,6 +253,23 @@ func initialSanitizedFieldNames() wildcard.Matchers {
 	return configutil.ParseWildcardPatternsEnv(envSanitizeFieldNames, defaultSanitizedFieldNames)
 }
 
+func initContinuationStrategy() (string, error) {
+	value := os.Getenv(envContinuationStrategy)
+	if value == "" {
+		return defaultContinuationStrategy, nil
+	}
+	return parseContinuationStrategy(value)
+}
+
+func parseContinuationStrategy(value string) (string, error) {
+	switch value {
+	case "continue", "restart", "restart_external":
+		return value, nil
+	default:
+		return "", fmt.Errorf("unknown continuation strategy: %s", value)
+	}
+}
```

Comment:
```suggestion
func validateContinuationStrategy(value string) error {
	switch value {
	case "continue", "restart", "restart_external":
		return nil
	default:
		return fmt.Errorf("unknown continuation strategy: %s", value)
	}
}
```

Seems a bit unnecessary to return the value that's passed in.

Reply:
Ah :facepalm: , good catch!